src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8715,6 +8715,7 @@ enum Precedence : uint
             Equality,
             Relational,
             Shift,
+            Range,
```

Comment:
>Range [](start = 12, length = 5)

The proposed associativity from the LDM is that the subexpressions of a range operator are at the Additive level, meaning `1..2..3` must be a syntax error. I suspect that isn't implemented here.

Reply:
No, likely not - I'll add a note to this PR's description calling it out as explicitly not-implemented (the nonassociativity, that is - precedence is still an open question)

> The proposed associativity from the LDM

I'm not sure what you mean here - the LDM hasn't met on this topic (and if they have, I am very concerned that I have not heard about it)

Reply:
The LDM specifically decided that the range operator will not be associative (i.e. neither right nor left). The subexpressions on both sides will be of the next precedence level.